### DISCUSSION TITLE
Turbulence_intensity varies with wind speed and wind direction

### DISCUSSION THREAD

**Discussion by redwantyr** [NatLabRockies/floris#735] [Q&A]
Hi,
the turbulence _intensity varies with wind speed and wind direction,  here is the code I'm using in floris

```python
time = np.arange(10, 845380, 10.)
ws= np.genfromtxt("inputs/WS_MM_fi.csv")
wd =  np.genfromtxt("inputs/WD_MM_fi.csv")
TI = np.genfromtxt("inputs/TI_MM_fi.csv")
fi.reinitialize(wind_directions=wd, wind_speeds=ws, turbulence_intensity= TI, time_series=True)
```
eventhough I used the recommended solution in #714, I'm getting the following error 

```
TypeError: only size-1 arrays can be converted to Python scalars
```

how to solve this problem?

regards,

**Comment by Bartdoekemeijer** [comment]
Hi @redwantyr. In FLORIS, the turbulence intensity is a single value that is the same for all wind directions and wind speeds, currently. Expanding it to be speed- and direction-dependent is tackled in #714.

**Reply by Bartdoekemeijer** [reply]
Oh my apologies, I didn't fully read your message correctly. The function from #714 probably just does not work with the `time_series` functionality.
